mirgecom/symbolic_fluid.py

```diff
@@ -0,0 +1,95 @@
+""":mod:`mirgecom.symbolic_fluid` provides symbolic versions of fluid constructs.
+
+Symbolic fluxes
+^^^^^^^^^^^^^^^
+.. autofunction:: sym_inviscid_flux
+.. autofunction:: sym_viscous_flux
+.. autofunction:: sym_diffusive_flux
+.. autofunction:: sym_heat_flux
+.. autofunction:: sym_viscous_stress_tensor
+
+Symbolic fluid operators
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. autofunction:: sym_euler
+.. autofunction:: sym_ns
+"""
+import numpy as np
+import numpy.random
```

Comment:
some unused imports.  address as you see fit; just a minor point.

Reply:
Fixed